src/account/accountActions.js

```diff
@@ -57,15 +57,19 @@ const loginSuccessPlain = (realm: URL, email: string, apiKey: string): AllAccoun
 });
 
 export const loginSuccess =
-  (realm: URL, email: string, apiKey: string): ThunkAction<Promise<void>> =>
-  async (dispatch, getState) => {
+  (realm: URL, email: string, apiKey: string): GlobalThunkAction<Promise<void>> =>
+  async (dispatch, getState, { activeAccountDispatch }) => {
     NavigationService.dispatch(resetToMainTabs());
     dispatch(loginSuccessPlain(realm, email, apiKey));
 
-    await dispatch(registerAndStartPolling());
+    // Now dispatch some actions on the new, post-login active account.
+    // Because we just dispatched `loginSuccessPlain`, that new account is
+    // now the active account, so `activeAccountDispatch` will act on it.
+
+    await activeAccountDispatch(registerAndStartPolling());
```

Comment:
nit:

    account actions types: Make loginSuccess thunk action a GlobalThunkAction

This does touch the actual logic, not only type annotations -- namely it starts using this `activeAccountDispatch` that this thunk can get from its extras object -- so "types" isn't right in the prefix, though "[nfc]" is.